### PR TITLE
Added 3 new methods to the API.

### PR DESCRIPTION
**Pull request by fillefilip8**
Added 3 new methods to the API.
"getPortedPlayers()"
"getPortedPlayersList()"
"getNonPortedPlayers()"

### CODE REVIEW

File: src/main/java/us/myles/ViaVersion/ViaVersionPlugin.java [hunk 1]
```diff
@@ -113,6 +114,22 @@ public boolean isDebug() {
         return this.debug;
     }
 
+    @Override
+    public Map<UUID, ConnectionInfo> getPortedPlayers() {
+        return portedPlayers;
```
Comment (Paulomart) [inline src/main/java/us/myles/ViaVersion/ViaVersionPlugin.java]: make it unmodifiable

**Comment by fillefilip8** [comment]
U mean make it final?

File: src/main/java/us/myles/ViaVersion/ViaVersionPlugin.java (hunk 1 above)
Comment (Paulomart) [inline src/main/java/us/myles/ViaVersion/ViaVersionPlugin.java]: `Collections.unmodifiableMap(m)`

**Comment by fillefilip8** [comment]
I made them unmodifiable now.

**Comment by HugoDaBosss** [comment]
Looks good. Is it safe to make the connectioninfo accessible by the API?
And maybe add a another method to just get a list of the UUIDs for the ported players.

**Comment by HugoDaBosss** [comment]
Yeah, this is good now. Will test it and then merge.

**Comment by Paulomart** [comment]
> Looks good. Is it safe to make the connectioninfo accessible by the API?
> And maybe add a another method to just get a list of the UUIDs for the ported players.

Could always use getPortedPlayers().getKeys() another method is not needed.

**Comment by HugoDaBosss** [comment]
Ofcourse it is usable, but now it's more convenient.

**Comment by Paulomart** [comment]
No. Now its uses more performance

**Pull request closed without merging**